stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/PageWideCard.jsx

```diff
@@ -10,21 +10,19 @@ export const PageWideCard = ({
   label, title, paragraph, button,
 }) => {
   useEffect(() => {
-    viewport('.wide-card__tag');
+    viewport('.wide-card h6');
     viewport('.wide-card__description');
   }, []);
   return (
     <div className="wide-card">
-      <div className="wide-card__tag left-right">{label}</div>
+      <Heading className="left-right" type="6" label={label} />
       <div className="grid-x wide-card__wrapper">
         <div className="cell medium-6 wide-card__description left-right">
-          <div className="wide-card__summary">
-            <a href="#">
-              <Heading type="4" label={title} />
-              <P label={paragraph} />
-            </a>
-            <CtaButton label1={button} />
-          </div>
+          <a href="#">
+            <Heading type="4" label={title} />
+            <P label={paragraph} />
+          </a>
+          <CtaButton label1={button} for_primary="arrow" />
         </div>
         <a className="cell medium-6 wide-card__image" href="#">
```

Comment:
@pallavikadam-srijan kindly wrap the anchor tag in a div tag and move the classes to the wrapping div, since anchor tag is an inline element and not a block level element.